src/common/business/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './low-object';
```

Comment:
All this common should have their respective specs?

Reply:
Tests added